.github/dependabot.yml

```diff
@@ -22,25 +22,22 @@ updates:
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: "weekly"
-      day: "sunday"
+      interval: "daily"
```

Comment:
maybe we should run this on Sunday + Thursday or so? My main concern is that dependabot updates become frequent and therefore spam us

Reply:
Why would it spam us? It will trigger at most upgrade across one release per dependency. Currently, I see upgrades across multiple releases, which is mode difficult to review.

Reply:
There is value in both of them, if you do weekly you sometimes skip a patch release (because they found an issue after a release, and did a follow-up patch release). If you run daily, you'll also catch those broken releases. I've created this PR, but I can go both ways.